dependency/actions/compile/bionic.Dockerfile

```diff
@@ -1,73 +1,75 @@
-FROM ubuntu:18.04
+FROM paketobuildpacks/build-bionic-full
 
-RUN apt-get update && apt-get install -y \
-      automake \
-      build-essential \
-      bundler \
-      cmake \
-      curl \
-      firebird-dev \
-      gcc \
-      git \
-      libargon2-0-dev \
-      libaspell-dev \
-      libbz2-dev \
-      libc-client2007e-dev \
-      libcurl4-openssl-dev \
-      libedit-dev \
-      libenchant-dev \
-      libexpat1-dev \
-      libgd-dev \
-      libgdbm-dev \
-      libgeoip-dev \
-      libgmp-dev \
-      libgpgme11-dev \
-      libjpeg-dev \
-      libkrb5-dev \
-      libldap2-dev \
-      libmagickcore-dev \
-      libmagickwand-dev \
-      libmaxminddb-dev \
-      libmcrypt-dev \
-      libmemcached-dev \
-      libonig-dev \
-      libpng-dev \
-      libpq-dev \
-      libpspell-dev \
-      librecode-dev \
-      libsasl2-dev \
-      libsnmp-dev \
-      libsqlite3-dev \
-      libsqlite3-dev \
-      libssh2-1-dev \
-      libssl-dev \
-      libtidy-dev \
-      libtool \
-      libwebp-dev \
-      libxml2-dev \
-      libxml2-dev \
-      libxslt1-dev \
-      libyaml-dev \
-      libzip-dev \
-      libzookeeper-mt-dev \
-      make \
-      pkg-config \
-      rbenv \
-      ruby \
-      ruby-dev \
-      rubygems \
-      snmp-mibs-downloader \
-      sqlite3 \
-      sudo \
-      unixodbc-dev
+ENV DEBIAN_FRONTEND noninteractive
 
-ADD ./extensions-manifests /tmp/extensions-manifests
+ARG cnb_uid=0
+ARG cnb_gid=0
 
-ADD ./binary-builder /binary-builder
-WORKDIR /binary-builder
+USER ${cnb_uid}:${cnb_gid}
 
-RUN bundle install
+RUN apt-get update && \
+  apt-get -y install \
+  autoconf \
+  automake \
+  bison \
+  build-essential \
+  cmake \
+  curl \
+  firebird-dev \
```

Comment:
If we're compiling on the stack image we shouldn't need to install all of these things because a good number of them should already be there. There may be a few we still need to install

Reply:
I removed packages that do not need to be reinstalled but some remained.